installation/nuget/nuget-keys.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Setting Up in .NET CLI
```

Comment:
I suggest keeping the title aimed at CI as it was in the original article as the focus of the article is CI and NuGet keys and not so much the .NET CLI, which is the instrument to download NuGet packages.